modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -228,14 +228,15 @@ export default class Tile3DLayer extends CompositeLayer {
 
   _makeSimpleMeshLayer(tileHeader, oldLayer) {
     const content = tileHeader.content;
-    const {attributes, modelMatrix, cartographicOrigin, texture} = content;
+    const {attributes, indices, modelMatrix, cartographicOrigin, texture} = content;
     const {getSimpleMeshLayerColor} = this.props;
 
     const geometry =
       (oldLayer && oldLayer.props.mesh) ||
       new Geometry({
         drawMode: GL.TRIANGLES,
-        attributes: getMeshGeometry(attributes)
+        attributes: getMeshGeometry(attributes),
+        indices
```

Comment:
Have we tested this change for datasets without indices?

Reply:
Yes. 1.6 works correctly